modules/oe_content_sub_entity/modules/oe_content_sub_entity_author/oe_content_sub_entity_author.module

```diff
@@ -48,3 +49,28 @@ function oe_content_sub_entity_author_node_prepare_form(EntityInterface $entity,
   ]);
   $entity->set('oe_authors', [$corporate_body]);
 }
+
+/**
+ * Implements hook_entity_delete().
+ *
+ * Delete author sub entities referencing a node being deleted.
```

Comment:
Delete author sub entities when the referenced node is being deleted. Author sub entity can reference Organisation or a Person node.

Reply:
Author entity has a field referencing nodes and we might add more node bundles later. I'd keep this comment as it is, doesn't change much. The if statement should be clear enough.